omnigibson/simulator.py

```diff
@@ -605,6 +606,8 @@ def add_ground_plane(self, floor_plane_visible=True, floor_plane_color=None):
                 # restitution=restitution,
             )
 
+            triangularize_mesh(lazy.pxr.UsdGeom.Mesh.Define(self.stage, plane.prim.GetChildren()[0].GetPath()))
```

Comment:
what is this needed for? curobo?

Reply:
Yes. The floor plane is not a triangular mesh, which breaks curobo's assumption.